ifsbench/config_utils.py

```diff
@@ -0,0 +1,25 @@
+import pathlib
+from typing import Dict, List, Union
+
+import yaml
+
+from ifsbench.config_mixin import CLASSNAME, ConfigMixin
+from ifsbench.data import ExtractHandler, NamelistHandler
+
+__all__ = ['read_yaml_config']
+
+_SUPPORTED_CONFIGS = {'ExtractHandler': ExtractHandler, 'NamelistHandler': NamelistHandler}
+
+def parse_config(config: Dict[str, Union[str, float, int, bool, List, None]]) -> Dict[str,ConfigMixin]:
```

Comment:
Docstring again, as public-facing API. 

The fact that this works as easy as this is amazing though! 🤩

Reply:
This actually wasn't supposed to be public, renamed to _parse_config. read_yaml is the public function, I added a doc string there